### PR TITLE
POCMF-1: deploy ampath-esm-root-config

### PR DESCRIPTION
**Pull request by madvincy**

### CODE REVIEW

File: src/root-config.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import { registerApplication, start } from "single-spa";
 import * as activityFunctions from "./routes";
 import "./register-shared-deps";
+import { registerAllCoreApplications } from "@openmrs/esm-root-config";
 
 export const coreApps = {
```
Comment (jonathandick) [inline src/root-config.ts]: nit: consider using another name other than coreApps to distinguish the "coreApps" in esm-root-config.  Perhaps ampathApps

Comment (joeldenning) [reply src/root-config.ts]: 👍 agreed

File: package.json [hunk 2]
```diff
@@ -48,6 +48,7 @@
     "webpack-dev-server": "^3.7.2"
   },
   "dependencies": {
+    "@openmrs/esm-root-config": "^1.0.0",
```
Comment (joeldenning) [inline package.json]: After my first PR review of this code, I released [@openmrs/esm-root-config@1.1.0](https://github.com/openmrs/openmrs-esm-root-config/releases/tag/v1.1.0). Please upgrade to that version. To do so:

```bash
npm install --save @openmrs/esm-root-config
```

File: src/root-config.ts (hunk 1 above)
Comment (jonathandick) [reply src/root-config.ts]: Also, I would recommend using the same pattern as done for esm-root-config where ampathApps exist in a separate file along with the associated activity function. See https://github.com/openmrs/openmrs-esm-root-config/blob/master/src/single-spa-applications/core-applications.js

**Comment by fatmali** [comment]
lgtm as well

**Pull request merged**